### PR TITLE
added option to show rating count in category view

### PR DESCRIPTION
**Pull request by MaxBentz**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been documented
- [ ] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [ ] Plugin can be built

@plentymarkets/plentyshop

### CODE REVIEW

**Comment by MaxBentz** [comment]
@sene-plenty ready for review :)

File: resources/views/DataProvider/CategoryAverageRating.twig [hunk 1]
```diff
@@ -13,5 +13,8 @@
             {% endfor %}
         </div>
     </div>
+        {% if counts.ratingsCountTotal > 0 and options.showRatingCountInCategoryView %}
+            <span>({{ Twig.print('counts.ratingsCountTotal') }})</span>
```
Comment (stentrop) [inline resources/views/DataProvider/CategoryAverageRating.twig:17]: The TwigBuilder is wrong here. No output is generated.

```suggestion
            <span>({{ counts.ratingsCountTotal }})</span>
```

**Review by stentrop — CHANGES_REQUESTED** [review]

**Review by Tim-M-S — CHANGES_REQUESTED** [review]
image
If the option showRatingCountInCategoryView is active you have to set a class to the stars divs and offset them by x px

**Comment by MaxBentz** [comment]
Class has been added. New build have to be made

File: resources/css/main.scss [hunk 2]
```diff
@@ -469,6 +469,11 @@
         line-height: 22px;
         font-size: 17px;
     }
+    .feedback-count {
+        position: absolute;
+        left: 88px;
+        top: 1px;
```
Comment (Tim-M-S) [inline resources/css/main.scss:475]:
```suggestion
        top: -1px;
```

**Review by Tim-M-S — DISMISSED** [review]

File: resources/lang/en/Config.properties [hunk 3]
```diff
@@ -5,6 +5,7 @@ releaseAuthenticated=Only for logged-in users
 releaseAll=Yes
 allowGuestFeedbacks=Allow reviews by guests
 showEmptyRatingsInCategoryViewLabel=Show empty star ratings in the category view
+showRatingCountInCategoryViewLabel=Show number of article ratings in the category view
```
Comment (Tim-M-S) [inline resources/lang/en/Config.properties:8]:
```suggestion
showRatingCountInCategoryViewLabel=Show number of reviews in the category view
```

**Review by Tim-M-S — DISMISSED** [review]

File: resources/css/main.scss [hunk 4]
```diff
@@ -469,6 +469,11 @@
         line-height: 22px;
         font-size: 17px;
     }
+    .feedback-count {
+        position: absolute;
+        left: 88px;
```
Comment (Tim-M-S) [inline resources/css/main.scss:474]:
```suggestion
        right: -20px
```

**Review by stentrop — APPROVED** [review]

**Pull request merged**